clock/app/javascript/packs/application.js

```diff
@@ -26,4 +26,29 @@ $(document).ready(function() {
   $(".close").click(function() {
     $("#Errors").toggle("slow");
   });
-});
\ No newline at end of file
+});
+<<<<<<< HEAD
+=======
```

Comment:
you have some merge conflicts here, pls solve them first
this could be helpful
https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line